packages/react-axes/src/react-axes/cfcs/useReactive.ts

```diff
@@ -10,27 +10,27 @@ export function useReactive<
   Data = any,
   Events extends Record<string, any> = {},
   >(reactiveProps: ReactiveAdapter<Instance, State, Methods, Data, Events>): ReactReactiveResult<Instance, State, Methods, Events> {
+  const data = reactiveProps.data ? reactiveProps.data() : {} as any;
   const reactiveState = reactiveProps.state as any;
-  const names = keys<Record<string, any>>(reactiveState);
+  const names = keys<Record<string, any>>(data.axis);
```

Comment:
cfcs should be general purpose code. If this code is copied and pasted into other code, can it be used as it is?

How about changing the state of useAxes instead of changing it here?

```js
useReactive({
     state: assign(vainilla state, axis to state)
});

Reply:
Considering that the cfc code is general purpose code, Axis specific codes have been moved to useAxes.

Reply:
Also, it seems that the part that changes the axisoption to axisState can be provided as a common function in vanilla.

It seems to be used by all frameworks.

Reply:
Added `getInitialPos` to utils, so that all cfc code can use it when calculating initial state.